packages/story-editor/src/app/highlights/states.ts

```diff
@@ -22,147 +22,111 @@ import { STYLE_PANE_IDS } from '@googleforcreators/design-system';
 /**
  * Internal dependencies
  */
-import { INSERT, DOCUMENT, STYLE } from '../../components/sidebar';
+import {
+  INSERT,
+  DOCUMENT,
+  STYLE as STYLE_SIDEBAR,
+} from '../../components/sidebar/constants';
 import { PANE_IDS } from '../../components/library/paneIds';
-
-/**
- * Highlight state object
- *
- * @typedef {Highlight} Highlight The current state of editor highlights
- * @property {string} tab The ID of the Sidebar tab to highlight
- * @property {boolean} focus Whether there is a focusable element, use for accessibility
- */
-
-const keys = {
-  STORY_TITLE: 'STORY_TITLE',
-  ELEMENT_TOOLBAR_TOGGLE: 'ELEMENT_TOOLBAR_TOGGLE',
-
-  // Sidebar tabs
-  STYLE_PANE: 'STYLE_PANE',
-
-  // STYLE
-  ANIMATION: 'ANIMATION',
-  ASSISTIVE_TEXT: 'ASSISTIVE_TEXT',
-  CAPTIONS: 'CAPTIONS',
-  EXCERPT: 'EXCERPT',
-  LINK: 'LINK',
-  PAGE_BACKGROUND: 'PAGE_BACKGROUND',
-  POSTER: 'POSTER',
-  PUBLISHER_LOGO: 'PUBLISHER_LOGO',
-  VIDEO_A11Y_POSTER: 'VIDEO_A11Y_POSTER',
-  STYLE: 'STYLE',
-  FONT: 'FONT',
-  TEXT_COLOR: 'TEXT_COLOR',
-
-  // LIBRARY
-  MEDIA: 'MEDIA',
-  MEDIA3P: 'MEDIA3P',
-  TEXT_SET: 'TEXT',
-  PAGE_TEMPLATES: 'PAGE_TEMPLATES',
-
-  // DOCUMENT
-  PAGE_BACKGROUND_AUDIO: 'PAGE_BACKGROUND_AUDIO',
-};
+import { HighlightType } from '../../types/highlightsProvider';
```

Comment:
Hm, the enum of keys should probably still stay here -- since it has actual values and is not just a type.
I suppose that's one of the reasons why enums are not great -- that they change the code instead of just adding types. cc @barklund for thoughts here as the new enum-master.

Reply:
Elsewhere we have left the enum into where the constants were before, for consistency, would be better to have it where before.

Reply:
_The Master of Enums_ henceforth declares, that we should keep enums in place of the constants they're replacing, because they (as @miina says) are transpiled to actual code and aren't stripped when type-checking. Also, this is what we've done elsewhere 🤷.